patches/tModLoader/Terraria/ModLoader/ModTile.cs

```diff
@@ -190,14 +214,15 @@ public virtual bool CanDrop(int i, int j)
-	/// Allows you to customize the items the tile at the given coordinates drops.
-	/// <br/> By default, this method will intelligently decide on a single item drop based on <see cref="ModBlockType.ItemDrop"/>, the tile style, and associated <see cref="TileObjectData"/> if it exists.
-	/// <br/> If <see cref="ModBlockType.ItemDrop"/> has a non-zero value, it will be used as the item to drop. If -1, no item will drop.
-	/// <br/> Otherwise, the dropped item will be the item type of the loaded item with <see cref="Item.createTile"/> and <see cref="Item.placeStyle"/> matching the type and style of the Tile. If the specific <see cref="Item.placeStyle"/> is not found, the decision will fall back to a <see cref="Item.placeStyle"/> of 0.
-	/// <br/> Detecting the tile style is only reliable for tiles with an associated <see cref="TileObjectData"/>, so tiles using a manual tile style approach need to override this method. Once the style is calculated from the tile frame data, <c>TileLoader.GetItemDropFromTypeAndStyle(Type, style)</c> can be used to retrieve the associated item drop. 
-	/// <br/> This existing logic should cover 99% of use cases, meaning that overriding this method should only be necessary in extremely unique tiles, such as tiles dropping multiple items, tiles dropping items with custom data, or tiles with custom tile style code.
-	/// <br/> For tiles dropping multiple items, or dropping items that need custom data, override this method. Use <c>yield return new Item(ItemTypeHere);</c> for each spawned item. 
-	/// <br/> Use <see cref="CanDrop"/> to conditionally prevent any item drops. Use <see cref="KillMultiTile(int, int, int, int)"/> or <see cref="KillTile(int, int, ref bool, ref bool, ref bool)"/> for other logic such as cleaning up TileEntities or killing chests or signs.
+	/// Allows customization of the items the tile at the given coordinates drops.<br/>
+	/// By default, this method will intelligently decide on a single item drop.<br/><br/>
+	/// For tiles without an associated <see cref="TileObjectData"/>, the item with <see cref="Item.createTile"/> matching this tile type will be dropped.<br/>
+	/// For tiles with an associated <see cref="TileObjectData"/>, the item with <see cref="Item.createTile"/> matching this tile type and <see cref="Item.placeStyle"/> matching the tile style of this tile will be dropped.<br/>
+	/// <see cref="ModTile.RegisterItemDrop(int, int[])"/> can be used to manually register item drops for tile styles with no corresponding item. It can also be used to register a fallback item, which will be dropped if no suitable item is found.<br/><br/>
+	/// Detecting the tile style is only reliable for tiles with an associated <see cref="TileObjectData"/>, so tiles using a manual tile style approach need to override this method. Once the style is calculated from the tile frame data, <c>TileLoader.GetItemDropFromTypeAndStyle(Type, style)</c> can be used to retrieve the associated item drop.<br/><br/>
```

Comment:
Maybe move this line to the bottom, and simplify. Something like `The style based drop logic is based on TileObjectData. If a tile has custom 'styles' but still wants to make use of <c>RegisterItemDrop<c>, <c>TileLoader.GetItemDropFromTypeAndStyle(Type, style)</c> can be used to retrieve the associated item drop`